<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Methods | User Guide</title>
  <meta name="description" content="User guide of an online tool for proteomics data normalization and batch effect correction" />
  <meta name="generator" content="bookdown 0.12 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Methods | User Guide" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="User guide of an online tool for proteomics data normalization and batch effect correction" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Methods | User Guide" />
  
  <meta name="twitter:description" content="User guide of an online tool for proteomics data normalization and batch effect correction" />
  

<meta name="author" content="Gang Peng" />


<meta name="date" content="2019-08-15" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="dataformat.html">
<link rel="next" href="output.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/kePrint-0.0.1/kePrint.js"></script>



<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">User Guide</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a><ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#general-illustration"><i class="fa fa-check"></i><b>0.1</b> General Illustration</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="dataformat.html"><a href="dataformat.html"><i class="fa fa-check"></i><b>2</b> Input Data Format</a><ul>
<li class="chapter" data-level="2.1" data-path="dataformat.html"><a href="dataformat.html#protein-file"><i class="fa fa-check"></i><b>2.1</b> Protein File</a></li>
<li class="chapter" data-level="2.2" data-path="dataformat.html"><a href="dataformat.html#df-sample"><i class="fa fa-check"></i><b>2.2</b> Sample Information File</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>3</b> Methods</a><ul>
<li class="chapter" data-level="3.1" data-path="methods.html"><a href="methods.html#single-batch"><i class="fa fa-check"></i><b>3.1</b> Single Batch</a><ul>
<li class="chapter" data-level="3.1.1" data-path="methods.html"><a href="methods.html#sample-loading"><i class="fa fa-check"></i><b>3.1.1</b> Sample Loading</a></li>
<li class="chapter" data-level="3.1.2" data-path="methods.html"><a href="methods.html#median"><i class="fa fa-check"></i><b>3.1.2</b> Median</a></li>
<li class="chapter" data-level="3.1.3" data-path="methods.html"><a href="methods.html#quantile"><i class="fa fa-check"></i><b>3.1.3</b> Quantile</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="methods.html"><a href="methods.html#multiple-batches"><i class="fa fa-check"></i><b>3.2</b> Multiple Batches</a><ul>
<li class="chapter" data-level="3.2.1" data-path="methods.html"><a href="methods.html#combat"><i class="fa fa-check"></i><b>3.2.1</b> ComBat</a></li>
<li class="chapter" data-level="3.2.2" data-path="methods.html"><a href="methods.html#limma"><i class="fa fa-check"></i><b>3.2.2</b> Limma</a></li>
<li class="chapter" data-level="3.2.3" data-path="methods.html"><a href="methods.html#irs"><i class="fa fa-check"></i><b>3.2.3</b> IRS</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="output.html"><a href="output.html"><i class="fa fa-check"></i><b>4</b> Output Results</a><ul>
<li class="chapter" data-level="4.1" data-path="output.html"><a href="output.html#boxplot"><i class="fa fa-check"></i><b>4.1</b> Boxplot</a></li>
<li class="chapter" data-level="4.2" data-path="output.html"><a href="output.html#coefficient-of-variation-cv"><i class="fa fa-check"></i><b>4.2</b> Coefficient of Variation (CV)</a></li>
<li class="chapter" data-level="4.3" data-path="output.html"><a href="output.html#sample-clustering"><i class="fa fa-check"></i><b>4.3</b> Sample Clustering</a></li>
<li class="chapter" data-level="4.4" data-path="output.html"><a href="output.html#how-to-choose-normalization-and-batch-effect-correction-method"><i class="fa fa-check"></i><b>4.4</b> How to Choose Normalization and Batch Effect Correction Method</a><ul>
<li class="chapter" data-level="4.4.1" data-path="output.html"><a href="output.html#boxplot-1"><i class="fa fa-check"></i><b>4.4.1</b> Boxplot</a></li>
<li class="chapter" data-level="4.4.2" data-path="output.html"><a href="output.html#coefficient-of-variation-cv-1"><i class="fa fa-check"></i><b>4.4.2</b> Coefficient of Variation (CV)</a></li>
<li class="chapter" data-level="4.4.3" data-path="output.html"><a href="output.html#sample-clustering-1"><i class="fa fa-check"></i><b>4.4.3</b> Sample Clustering</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="output.html"><a href="output.html#download-results"><i class="fa fa-check"></i><b>4.5</b> Download Results</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">User Guide</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="methods" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> Methods</h1>
<p>If there is only one batch in the input data, the online tool only take the data normalization step. If there are multiple batches in the input data, the online tool normalize the data in each batch first and then take the batch effect correction. Figure <a href="methods.html#fig:fig-workflow">3.1</a> shows the the general workflow of the tool.</p>
<div class="figure" style="text-align: center"><span id="fig:fig-workflow"></span>
<img src="images/PTNormFlowChart.png" alt="Workflow of the online tool" width="50%" />
<p class="caption">
Figure 3.1: Workflow of the online tool
</p>
</div>
<div id="single-batch" class="section level2">
<h2><span class="header-section-number">3.1</span> Single Batch</h2>
<p>If the input data is in a single batch, the online tool only takes the normalization step. The following normalization methods are implemented in the online tool.</p>
<div id="sample-loading" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Sample Loading</h3>
<p>This method assumes that the summation of abundance for all the proteins is the same for all the sample.</p>
</div>
<div id="median" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Median</h3>
<p>The median normalization is based on the assumption that the median of protein abundance is the sample for all the samples.</p>
</div>
<div id="quantile" class="section level3">
<h3><span class="header-section-number">3.1.3</span> Quantile</h3>
<p>In quantile normalization, the quantile of each sample is replaced by the mean of the same quantile of all the sample <span class="citation">(Bolstad et al. <a href="#ref-bolstad2003comparison">2003</a>)</span>.</p>
</div>
</div>
<div id="multiple-batches" class="section level2">
<h2><span class="header-section-number">3.2</span> Multiple Batches</h2>
<p>If the input data contains samples from multiple batches, the online tool will take normalization step for each batch first and then remove the batch effect between different batches. Now we have implemented the following batch effect correction methods in the tool.</p>
<div id="combat" class="section level3">
<h3><span class="header-section-number">3.2.1</span> ComBat</h3>
<p>We used ComBat method in R package <em>sva</em>. The details of the method can be file in the paper <span class="citation">(Johnson, Li, and Rabinovic <a href="#ref-johnson2007adjusting">2007</a>)</span>.</p>
</div>
<div id="limma" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Limma</h3>
<p>Function <strong>removeBatchEffect</strong> in R package <em>limma</em> <span class="citation">(Ritchie et al. <a href="#ref-ritchie2015limma">2015</a>)</span> is used to remove the batch effect.</p>
</div>
<div id="irs" class="section level3">
<h3><span class="header-section-number">3.2.3</span> IRS</h3>
<p>This method is based on the same pooling sample across different batches. It assumes that the pooling samples have the same protein abundance at different batches and calculates a scaling factor from this assumption. The details of this method can be found at <span class="citation">(Plubell et al. <a href="#ref-plubell2017extended">2017</a>)</span>.</p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-bolstad2003comparison">
<p>Bolstad, Benjamin M, Rafael A Irizarry, Magnus Åstrand, and Terence P. Speed. 2003. “A Comparison of Normalization Methods for High Density Oligonucleotide Array Data Based on Variance and Bias.” <em>Bioinformatics</em> 19 (2). Oxford University Press: 185–93.</p>
</div>
<div id="ref-johnson2007adjusting">
<p>Johnson, W Evan, Cheng Li, and Ariel Rabinovic. 2007. “Adjusting Batch Effects in Microarray Expression Data Using Empirical Bayes Methods.” <em>Biostatistics</em> 8 (1). Oxford University Press: 118–27.</p>
</div>
<div id="ref-plubell2017extended">
<p>Plubell, Deanna L, Phillip A Wilmarth, Yuqi Zhao, Alexandra M Fenton, Jessica Minnier, Ashok P Reddy, John Klimek, Xia Yang, Larry L David, and Nathalie Pamir. 2017. “Extended Multiplexing of Tandem Mass Tags (Tmt) Labeling Reveals Age and High Fat Diet Specific Proteome Changes in Mouse Epididymal Adipose Tissue.” <em>Molecular &amp; Cellular Proteomics</em> 16 (5). ASBMB: 873–90.</p>
</div>
<div id="ref-ritchie2015limma">
<p>Ritchie, Matthew E, Belinda Phipson, Di Wu, Yifang Hu, Charity W Law, Wei Shi, and Gordon K Smyth. 2015. “Limma Powers Differential Expression Analyses for Rna-Sequencing and Microarray Studies.” <em>Nucleic Acids Research</em> 43 (7). Oxford University Press: e47–e47.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="dataformat.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="output.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
